content/en/docs/installing-on-linux/introduction/air-gapped-installation.md

```diff
@@ -215,7 +215,7 @@ As you install KubeSphere and Kubernetes on Linux, you need to prepare an image
 
    {{< notice note >}}
 
-   - You can change the Kubernetes version downloaded based on your needs. Recommended Kubernetes versions for KubeSphere v3.1.0: v1.17.9, v1.18.8, v1.19.8 and v1.20.4. If you do not specify a Kubernetes version, KubeKey will install Kubernetes v1.19.8 by default. For more information about supported Kubernetes versions, see [Support Matrix](../kubekey/#support-matrix).
+   - You can change the Kubernetes version downloaded based on your needs. Recommended Kubernetes versions for KubeSphere v3.1.1: v1.17.9, v1.18.8, v1.19.8, v1.20.4, and v1.20.6. If you do not specify a Kubernetes version, KubeKey will install Kubernetes v1.19.8 by default. For more information about supported Kubernetes versions, see [Support Matrix](../kubekey/#support-matrix).
```

Comment:
I added `v1.20.6` to the recommended Kubernetes versions for KubeSphere v3.1.1. Please confirm whether this is correct.